packages/hash/shared/src/queries/entity.queries.ts

```diff
@@ -170,15 +189,13 @@ export const updateEntity = gql`
       properties: $properties
     ) {
       __typename
-      id
```

Comment:
We shouldn't be using these ambiguous `id` fields and I want to remove them entirely from the API in a future PR ([task](https://app.asana.com/0/1200211978612931/1202024133311850/f))